stupidns/stupidns.go

```
@@ -0,0 +1,84 @@
+	s.responseQueue = s.responseQueue[1:]
+}
+
+func Create(addr string) StupiDNS {
+	var s StupiDNS
+	s.srv = dns.Server{}
+	s.srv.Net = "tcp"
+	s.srv.Addr = addr
+
+	s.responseQueue = make([]*dns.Msg, 0)
+
+	return s
```

golangci-lint (1.62.2)

copylocks: return copies lock value: github.com/zluudg/tdns/stupidns.StupiDNS contains github.com/miekg/dns.Server contains sync.RWMutex

(govet)